### DISCUSSION TITLE
Sesiones de hoy

### DISCUSSION THREAD

**Discussion by mmasias** [mmasias/24-25-IdSw1#131] [Announcements]
Hoy dedicaremos la sesión a los grupos que quieran hacer revisión presencial de los avances de sus trabajos: estaré disponible en mi despacho para dichas revisiones.

**Comment by ghost** [comment]
Buen día Masias, en el repo de Diego Moys he solicitado revisión de unos cambios que pediste. Podrías verlos y revisarlo por favor.
